# Spanish translation of django-otp.
# Copyright (C) Listed translators
# This file is distributed under the same license as the django-otp package.
# Miguel González <migonzalvar@gmail.com>, 2022
#
msgid ""
msgstr ""
"Project-Id-Version: django-otp master\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2024-08-13 16:08+0100\n"
"PO-Revision-Date: 2022-11-09 19:43+0100\n"
"Last-Translator: \n"
"Language-Team: \n"
"Language: es\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: myenv/lib/python3.12/site-packages/django_otp/forms.py:60
msgid "Please enter your OTP token."
msgstr "Por favor introduzca su token OTP."

#: myenv/lib/python3.12/site-packages/django_otp/forms.py:61
#, python-brace-format
msgid "Error generating challenge: {0}"
msgstr "Error al generar el reto: {0}"

#: myenv/lib/python3.12/site-packages/django_otp/forms.py:62
msgid "The selected OTP device is not interactive"
msgstr "EL dispositivo OTP seleccionado no es interactivo"

#: myenv/lib/python3.12/site-packages/django_otp/forms.py:63
#, python-brace-format
msgid "OTP Challenge: {0}"
msgstr "Reto OTP: {0}"

#: myenv/lib/python3.12/site-packages/django_otp/forms.py:65
msgid "Invalid token. Please make sure you have entered it correctly."
msgstr ""
"Token no válido. Por favor asegúrese de que lo ha introducido correctamente."

#: myenv/lib/python3.12/site-packages/django_otp/forms.py:68
#, python-format
msgid ""
"Verification temporarily disabled because of %(failure_count)d failed "
"attempt, please try again soon."
msgid_plural ""
"Verification temporarily disabled because of %(failure_count)d failed "
"attempts, please try again soon."
msgstr[0] ""
"La verificación está temporalmente deshabilitada porque ha habido "
"%(failure_count)d intento fallido, por favor inténtelo de nuevo más tarde."
msgstr[1] ""
"La verificación está temporalmente deshabilitada porque ha habido "
"%(failure_count)d intentos fallidos, por favor inténtelo de nuevo más tarde."

#: myenv/lib/python3.12/site-packages/django_otp/forms.py:73
msgid "Verification of the token is currently disabled"
msgstr "La verificación del token está desactivada en este momento"

#: myenv/lib/python3.12/site-packages/django_otp/plugins/otp_email/models.py:105
msgid "sent by email"
msgstr ""

#: myenv/lib/python3.12/site-packages/django_otp/templates/otp/admin111/login.html:38
msgid "Please correct the error below."
msgstr "Por favor corrija el error abajo indicado."

#: myenv/lib/python3.12/site-packages/django_otp/templates/otp/admin111/login.html:38
msgid "Please correct the errors below."
msgstr "Por favor corrija los errores abajo indicados."

#: myenv/lib/python3.12/site-packages/django_otp/templates/otp/admin111/login.html:54
#, python-format
msgid ""
"You are authenticated as %(username)s, but are not authorized to access this "
"page. Would you like to login to a different account?"
msgstr ""
"Usted está autenticado como %(username)s, pero no está autorizado a acceder "
"a esta página. ¿Desea iniciar sesión con una cuenta diferente?"

#: myenv/lib/python3.12/site-packages/django_otp/templates/otp/admin111/login.html:74
msgid "OTP Device:"
msgstr "Dispositivo OTP:"

#: myenv/lib/python3.12/site-packages/django_otp/templates/otp/admin111/login.html:79
msgid "OTP Token:"
msgstr "Token OTP:"

#: myenv/lib/python3.12/site-packages/django_otp/templates/otp/admin111/login.html:84
msgid "Forgotten your password or username?"
msgstr "¿Ha olvidado la contraseña o el nombre de usuario?"

#: myenv/lib/python3.12/site-packages/django_otp/templates/otp/admin111/login.html:88
msgid "Log in"
msgstr "Iniciar sesión"

#: myenv/lib/python3.12/site-packages/django_otp/templates/otp/admin111/login.html:90
msgid "Get OTP Challenge"
msgstr "Obtenga reto OTP"
